README.md

```diff
@@ -63,17 +63,22 @@ logging.basicConfig(level=logging.DEBUG)
 
 ### Get api_secret
 api_secret is unique for each and every account. You need to enable api trading and get api_secret from alice blue.
-Please [contact](https://www.aliceblueonline.com/contact-us/) alice blue for getting api_secret.
+1. Please [contact](https://www.aliceblueonline.com/contact-us/) alice blue to get access to api.
+2. After you get a response from alice blue, login to [developer console](https://develop-api.aliceblueonline.com/dashboard)
+3. Click on 'Create App'
+4. Enter 'App Name' as you like. Enter 'Redirect URI' as 'http://develop-api.aliceblueonline.com/'
```

Comment:
Change this to `https://ant.aliceblueonline.com/plugin/callback`.
That is the default redirect_url of the library.